src/main/java/com/hermesworld/ais/galapagos/kafka/impl/ConnectedKafkaCluster.java

```diff
@@ -77,14 +77,18 @@ public String getId() {
 
     @Override
     public CompletableFuture<Void> updateUserAcls(KafkaUser user) {
+
```

Comment:
Lots of empty lines added here in the class :-) Please remove them all again, as there is no real change to this class.